schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,522 @@
+<br><br>
+
+### is_primary_data
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>is_primary_data</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>bool</code>. This MUST be <code>True</code> if this is the canonical instance of this cellular observation and <code>False</code> if not. This is commonly <code>False</code> for meta-analyses reusing data or for secondary views of data.
+        </td>
+    </tr>
+</tbody></table>
+<br><br>
+
+### organism_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>organism_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. This MUST be either <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i> or <code>"NCBITaxon:10090"</code> for <i>Mus musculus</i>.
+        </td>
+    </tr>
+</tbody></table>
+<br><br>
+
+### sex_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>sex_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. This MUST be a child of <a href="http://purl.obolibrary.org/obo/PATO_0001894">PATO:0001894</a> for  <i>phenotypic sex</i>.
+        </td>
+    </tr>
+</tbody></table>
+<br><br>
+
+### tissue_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>tissue_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator<td>
+    </tr>
+   <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. MUST be the term that best describes the tissue that this cell was derived from, depending on the type of biological sample:
+          <br><br>
+          <table>
+          <thead>
+          <tr>
+          <th>For</th>
+          <th>Use</th>
+          </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Cell Culture</td>
+              <td>MUST be a CL term appended with <code>" (cell culture)"</code> (e.g. <a href="http://purl.obolibrary.org/obo/CL_0000057"><code>"CL:0000057</a> (cell culture)"</code> for the WTC-11 cell line)</td>
+            </tr>
+            <tr>
+              <td>Organoid</td>
+              <td>MUST be an UBERON term appended with <code>" (organoid)"</code> (e.g. <a href="http://purl.obolibrary.org/obo/UBERON_0000955"><code>"UBERON:0000955</a> (organoid)"</code> for a <i>brain organoid</i>)</td>
+            </tr>
+            <tr>
+              <td><a href="https://en.wikipedia.org/wiki/Flow_cytometry#Cell_sorting_by_flow_cytometry">Sorted cells</a> from a tissue</td>
+              <td>MUST be a CL term (e.g. <a href="http://purl.obolibrary.org/obo/CL_0000766"><code>"CL:0000766"</code></a> for sorted <i>myeloid leukocyte</i> cells)</i></td>
```

Comment:
This one worries me a little. First, 'sorting' may be too narrow as you may want to capture various methods of cell type enrichment/isolation
More importantly, if I enrich leukocytes from kidney tissue, I would follow these guidelines and submit CL:0000738. But a user querying the database for 'kidney data' would not receive my data.

Reply:
Good point. I reviewed the ontology and I think I picked a poor example, since immune cells are structured in the hematopoietic lineage, not tissues. I think it's still a good idea to include this line. Here's how I'd modify it:

For: Enriched, Sorted, or Isolated Cells from a tissue. 

use: MUST be an UBERON or CL term. SHOULD NOT use terms that do not capture the tissue of origin. (e.g. do not use CL_0000084, _T cell_ for CD3+ kidney cells. Use UBERON:0002113, for _Kidney_ instead. However, use CL:0000066, _Epithelial cells of the cervix_ for EPCAM+ cervical cells)

Tissue should be moved to the top - that's the most common use case. My fault for suggesting the rows in the wrong order.

Reply:
I like the suggested modifications.
(but lowercase term labels _kidney_ & _epithelial cells of the cervix_ to match the ontology labels)

Reply:
> Tissue should be moved to the top - that's the most common use case. My fault for suggesting the rows in the wrong order.

Not your fault. By design. I've been sorting most columns (keys).

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/d3b0364c33a02f10ac088da0805bab39e0d1cb5f - Tissue on top.